code/game/objects/items/food/misc.dm

```diff
@@ -1,6 +1,16 @@
 
 ////////////////////////////////////////////OTHER////////////////////////////////////////////
 
+/obj/item/food/cheese
```

Comment:
item/food/chese was defined after it's parents and it made me very uncomfortable